backend/src/index.ts

```
@@ -214,5 +217,13 @@ const shutdown = (signal: string) => {
   });
 };
 
+const io = new Server(httpServer, { /* options */ });
```

Biome (2.1.2)

[error] Shouldn't redeclare 'io'. Consider to delete it or rename it.

'io' is defined here:


(lint/suspicious/noRedeclare)